src/Compilers/CSharp/Portable/Declarations/DeclarationModifiers.cs

```diff
@@ -36,8 +36,12 @@ internal enum DeclarationModifiers : uint
         Async = 1 << 20,
         Ref = 1 << 21, // used only for structs
 
-        All = (1 << 23) - 1, // all modifiers
-        Unset = 1 << 23, // used when a modifiers value hasn't yet been computed
+
+        // PROTOTYPE(ft): leaving 22 free since required is using it
+        File = 1 << 23, // used only for structs
+
+        All = (1 << 24) - 1, // all modifiers
+        Unset = 1 << 24, // used when a modifiers value hasn't yet been computed
 
         AccessibilityMask = PrivateProtected | Private | Protected | Internal | ProtectedInternal | Public,
```

Comment:
Is `File` intended to be missing here? Consider adding a comment if that's intended

Reply:
Sure. It's true that we aren't treating file as an accessibility. Think of accessibility as controlling whether you're allowed to use something at a certain location. 'file' doesn't control that, it actually controls whether the thing *exists* at a certain location, i.e. like a scope does for a local variable.

Reply:
Makes sense. Thanks!